Gemfile

```diff
@@ -101,6 +101,8 @@ gem "rouge"
 
 gem "ransack"
 
+gem "timecop"
```

Comment:
Is there a way to prevent this from being loaded in production?

Reply:
Aha, I've just spotted the initializers

Reply:
Yeah, I will move it to sandbox, develop etc group

Reply:
Should be all environments except production.

Reply:
Even staging?

Reply:
Are there any other exceptions on staging where it's not an 100% match to production? If not, then no.